MAINTAINERS.md

```diff
@@ -0,0 +1,16 @@
+# MAINTAINERS
+
+## Current Maintainers
+
+_Listed in alphabetical order by first name_
+
+| Name | GitHub Username |
+| --- | --- |
+| Adam Reese | adamreese |
+| Ivan Towlson | itowlson |
```

Comment:
I don't think I should be on this.  My only involvement was to copy it from Finicky Whiskers.